.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: derisen
+
+---
+
+>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.
+
+## Library
+
+- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
+- [ ] `@azure/msal-browser@2.x.x`
+- [ ] `@azure/msal-angular@0.x.x`
+- [ ] `@azure/msal-angular@1.x.x`
+- [ ] `@azure/msal-angularjs@1.x.x`
```

Comment:
Probably don't want to include `@azure/msal-browser` here

Reply:
aah of course. done!